ibis/pandas/execution/window.py

```diff
@@ -142,12 +153,21 @@ def execute_window_op(
     pre_executed_scope = pre_execute(
         operand_op, *clients, scope=scope, aggcontext=aggcontext, **kwargs
     )
+    computed_localcontexts = compute_local_context(
```

Comment:
Why do we need to insert this inside `execute_window_op`? Didn't we have have this in `execute_until_in_scope`?

Reply:
According to the doc in this function, pre_execute is called manually here because otherwise we wouldn't relevant scope changes from the child operand since we're execution of that by hand. Thus compute_time_context should be also called before pre_execute to pass time context in

Reply:
In ``window.py`` there is a manual execute sequence and we need to inject this, and time contexts accordingly.